<launch>
    <arg name="node_name" />
    <arg name="target_model_name"  />
    <arg name="goal_pose_x"   />
    <arg name="goal_pose_y"  />
    <arg name="/path_topic"   />
    <arg name="/pose_stamped_topic"   />
    <arg name="/visualization_marker"   />
    <arg name="/visualization_marker_array"   />
    <node name = "$(arg node_name)" pkg = "orcawsdwa" type = "pub_sub" output = "screen">
     <param name="target_model_name" type="string" value="$(arg target_model_name)" />
     <param name="target_model/twist_linear_x" type="double" value="0.5" />
     <param name="target_model/twist_angular_z" type="double" value="1.0" />
     <param name="neighbor_distance" type="double" value="2.0" />
     <param name="time_horizon" type="double" value="2" />
     <param name="radius_" type="double" value="0.38" />  
     <param name="goal_pose_x" type="double" value="$(arg goal_pose_x)" />
     <param name="goal_pose_y" type="double" value="$(arg goal_pose_y)" />
     <param name="num" type="double" value="101" />
     <param name="time" type="double" value="0.1" />
     <param name="max_speed" type="double" value="10.0" />
     <param name="sample_num" type="double" value="200" />
     <param name="step" type="double" value="1" />
     <param name="size_" type="double" value="5.0" />
     <param name="ratio" type="double" value="0.2" />
     <param name="max_angular_speed" type="double" value="7" />
     <param name="max_linear_speed" type="double" value="7" />
     <remap from="/pose_stamped_topic" to="$(arg /pose_stamped_topic)" />
     <remap from="/path_topic" to="$(arg /path_topic)" />
     <remap from="/visualization_marker" to="$(arg /visualization_marker)" />
     <remap from="/visualization_marker_array" to="$(arg /visualization_marker_array)" />
    </node>

</launch>